_overviews/scala3-book/domain-modeling-fp.md

```diff
@@ -228,6 +324,9 @@ A first approach is to define the behavior---the functions---in a companion obje
 
 With this approach, in addition to the enumeration or case class you also define an equally named companion object that contains the behavior.
 
+{% tabs org_1 %}
+{% tab 'Scala 3 only' for=org_1 %}
```

Comment:
please could you also add a scala 2 tab that ports the enum to sealed trait and case objects

Reply:
I have no capability to give scala2 code.